test_runner/fixtures/neon_fixtures.py

```diff
@@ -2237,6 +2237,19 @@ def static_proxy(
         yield proxy
 
 
+@pytest.fixture(
+    scope="function",
+    params=[
+        "project",
+        "endpoint",
+    ],
+)
+def proxy_endpoint_option_name(request: FixtureRequest) -> str:
+    assert hasattr(request, "param")
+    assert isinstance(request.param, str)
+    return request.param
```

Comment:
To me putting this small piece in the fixtures library doesn't make much sense. It's very ad-hoc and hard to discover.

 I'd solve this using `pytest.mark.parametrize`:

```python
@pytest.mark.parametrize("name", ["endpoint", "project"])
def test_fizz_buzz(name: str):
    pass
```

Reply:
It was my initial implementation, btw. But I followed your idea from initial comment

Reply:
Sorry if I gave a wrong impression; this is what I had in mind at that time.